test/jest_integration/jwt_auth.test.ts

```diff
@@ -55,7 +55,7 @@ describe('start OpenSearch Dashboards server', () => {
           verbose: false,
         },
         opensearch: {
-          hosts: ['https://localhost:9200'],
+          hosts: ['https://[::1]:9200'],
```

Comment:
just being curious, is this also related to the node 20 bump? 👀

Reply:
ahhh nvm i just noticed this is not the upstream :/

Reply:
yea, I opened this up here for testing first. I'll port it over to the upstream when all checks pass.